docker-compose.yml

```diff
@@ -49,6 +49,7 @@ services:
         USER: "django"
         DJANGO_CONFIGURATION: "production"
         WITH_TESTS: "no"
+        OPENVINO_TOOLKIT: "no"
```

Comment:
@navalchand 
Hi, thanks for PR. Why do we need this declaration here?

Reply:
It would help staters easily build docker if they require auto annotation. Because i have struggled too as I was new to docker.